style.css

```diff
@@ -0,0 +1,339 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;600;700;800&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background: #fff;
+  color: #1a2236;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+p {
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  text-align: left;
+}
+
+h1 {
+  margin: 0;
+}
+
+h3 {
+  font-weight: 300;
+  font-size: 20px;
```

Comment:
It would be better to use responsive measurements such as em, rem, and percentages as they scale with the screen size and are better for Responsive Web Design (RWD).